todo/src/components/Todo.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { addTodo, toggleCompleted, deleteTodo, clearCompleted } from '../actions';
+
+class Todo extends React.Component {
+    state = {
```

Comment:
good job using the local state here for the single todo to make it easier to use